plugins/jspsych-video.js

```diff
@@ -115,12 +99,18 @@ jsPsych.plugins.video = (function() {
 
     display_element.innerHTML = video_html;
 
+    var video_blob = jsPsych.pluginAPI.getVideoBuffer(file_name);
+    if(!video_blob) throw(file_name + ' (video) not preloaded')
+
+    display_element.childNodes[0].src = video_blob;
```

Comment:
Can this fall back to the old way of doing it when `!video_blob` is `true`?

Reply:
exploring. May well work if I remove that throw.

Reply:
Nope. I just reintegrated some of the past code.